### PR TITLE
make DMI work on solaris

### LINKED ISSUE #539: no dmi information on solaris2
Hello!

I've installed chef-client 11.14.6 on Solaris 10 (x86 and sparc) with ohai 7.2.4. I'm trying to fetch DMI Information from our solaris systems, but only get an empty hash, when querying DMI information.

root@solaris-host:/ # ohai dmi
{

}

I observe the same behaviour while running chef-shell or chef-client.

Kind regards,
Lothar

### PR DESCRIPTION
**Pull request by thommay**
Fixes #539 

cc @chef/client-core

### CODE REVIEW

File: spec/unit/plugins/solaris2/dmi_spec.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
+#
+# License:: Apache License, Version 2.0
```
Comment (btm) [inline spec/unit/plugins/solaris2/dmi_spec.rb]: add a copyright notice (and author) please.

**Comment by thommay** [comment]
@btm thanks, done. we seem to be wildly inconsistent on this...

**Comment by btm** [comment]
was the https://ci.appveyor.com/project/chef/ohai/build/master-124 failure due to reusing the DMI_OUT constant?

:+1:

**Comment by thommay** [comment]
@btm yeah. doh!

**Comment by danielsdeleo** [comment]
LGTM, :+1: if travis and friends are happy

**Pull request merged**